stable/redis-ha/values.yaml

```diff
@@ -1,54 +1,86 @@
+    ## For all available options see http://download.redis.io/redis-stable/redis.conf
+    min-slaves-to-write: 1
+    min-slaves-max-lag: 5   # Value in seconds
+    maxmemory: "0"       # Max memory to use for each redis instance. Default is unlimited.
+    maxmemory-policy: "volatile-lru"  # Max memory policy to use for each redis instance. Default is volatile-lru.
+    # Determines if scheduled RDB backups are created. Default is false.
+    # Please note that local (on-disk) RDBs will still be created when re-syncing with a new slave. The only way to prevent this is to enable diskless replication.
+    save: "900 1"
+    # When enabled, directly sends the RDB over the wire to slaves, without using the disk as intermediate storage. Default is false.
+    repl-diskless-sync: "yes"
+    rdbcompression: "yes"
+    rdbchecksum: "yes"
+
+  resources:
     requests:
+
       memory: 200Mi
       cpu: 100m
     limits:
       memory: 700Mi
-  sentinel:
+
+## Sentinel specific configuration options
+sentinel:
+  port: 26379
+  quorum: 2
+  config:
+    ## Additional sentinel conf options can be added below
+    ## For all available options see http://download.redis.io/redis-stable/sentinel.conf
+    down-after-milliseconds: 10000
+    ## Failover timeout value in milliseconds
+    failover-timeout: 180000
+    parallel-syncs: 5
+
+  resources:
     requests:
       memory: 200Mi
       cpu: 100m
     limits:
       memory: 200Mi
 
-## Node labels, tolerations, and affinities for pod assignment
+securityContext:
+  runAsUser: 1000
+  fsGroup: 1000
+  runAsNonRoot: true
+
+## Node labels, affinity, and tolerations for pod assignment
 ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
 ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
 ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
-podAntiAffinity:
-  sentinel: soft
-  server: soft
+podAntiAffinity: soft
 
-## Redis image version
-redis_image: quay.io/smile/redis:4.0.8r0
-## replicas number for each component
-replicas:
-  servers: 3
-  sentinels: 3
-servers:
-  serviceType: ClusterIP  # [ClusterIP|LoadBalancer]
-  annotations: {}
-
-rbac:
-  # Specifies whether RBAC resources should be created
-  create: true
+podDisruptionBudget:
+  maxUnavailable: 1
 
-serviceAccount:
-  # Specifies whether a ServiceAccount should be created
-  create: true
-  # The name of the ServiceAccount to use.
-  # If not set and create is true, a name is generated using the fullname template
-  name:
-
-## Configures redis with AUTH (requirepass & masterauth conf params)
+# redisPassword:
 auth: false
 
-## Redis password
-## Defaults to a random 10-character alphanumeric string if not set and auth is true
-## ref: https://github.com/kubernetes/charts/blob/master/stable/redis-ha/templates/redis-auth-secret.yaml
-##
-## redisPassword:
+persistentVolume:
+  enabled: false
```

Comment:
As per the review guidelines https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md
"Persistence should be enabled by default"

Reply:
This is now resolved. Thanks!